apps/docs/pages/guides/platform/migrating-and-upgrading-projects.mdx

```diff
@@ -74,28 +74,13 @@ When upgrading, the Supabase platform will "right-size" your disk based on the c
 pg_upgrade does not support upgrading of databases containing reg\* data types referencing system OIDs.
 If you have created any objects that depend on the following extensions, you will need to recreate them after the upgrade.
 
-Extensions registering system OID-specific reg\* data types:
+#### Extensions
 
-- `plv8`
-- `plcoffee`
-- `plls`
+pg_upgrade does not currently support upgrading of databases using extensions older than the following versions:
```

Comment:
Alternative wording, though it might prove confusing for users skimming the docs, leading them to believe that projects using those specific versions can't be upgraded, rather than the reverse

```suggestion
pg_upgrade does not currently support upgrading of databases using extensions prior to following versions:
```